main/glossary/README.md

```diff
@@ -163,15 +156,15 @@ See [here](https://github.com/Agoric/agoric-sdk/blob/master/packages/SwingSet/do
 ## E()
 
 (Also referred to as *eventual send*) `E()` is a local "bridge" function that invokes methods on remote objects, for example
```

Comment:
We use `E` to send to local and remote objects. The key features are that the designated object *can be* remote, and that the invocation is always asynchronous, to be handled in its own later turn. The expression returns a promise for the invocation's result.